cluster/cluster_members.go

```diff
@@ -52,15 +52,15 @@ type InternalClusterMember struct {
 	Role           Role
 }
 
-// InternalClusterMemberFilter is used for filtering queries using generated methods.
-type InternalClusterMemberFilter struct {
+// CoreClusterMemberFilter is used for filtering queries using generated methods.
+type CoreClusterMemberFilter struct {
 	Address *string
 	Name    *string
 }
 
 // ToAPI returns the api struct for a ClusterMember database entity.
 // The cluster member's status will be reported as unreachable by default.
-func (c InternalClusterMember) ToAPI() (*internalTypes.ClusterMember, error) {
+func (c CoreClusterMember) ToAPI() (*internalTypes.ClusterMember, error) {
```

Comment:
As part of this should we also align the name on the Go level and rename `internal/rest/types` to `core/rest/types`?

Reply:
Nevermind, you have already done this change in one of the later commits.